backend/app/api/endpoints/adapter/attachments.py

```
@@ -88,6 +101,115 @@ def _build_content_disposition(filename: str) -> str:
+        payload = jwt.decode(
+            download_token,
+            settings.SECRET_KEY,
+            algorithms=[settings.ALGORITHM],
+        )
+    except JWTError as exc:
+        logger.warning(
+            "Invalid attachment download token: attachment_id=%s reason=%s",
+            attachment_id,
+            exc.__class__.__name__,
+        )
+        raise HTTPException(status_code=401, detail="Invalid download token")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)